plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -471,6 +526,13 @@ def unblockUser(self):
         return user.unblock()
 
 
+def sanitize_arguments(arguments):
+    for key, value in dict(arguments).items():
```

Comment:
Why not:
```suggestion
    for key, value in list(arguments.items()):
```
I think that's easier to read.